servers/physics_3d/soft_body_3d_sw.cpp

```diff
@@ -899,26 +902,64 @@ void SoftBody3DSW::add_velocity(const Vector3 &p_velocity) {
 }
 
 void SoftBody3DSW::apply_forces() {
-	if (pressure_coefficient < CMP_EPSILON) {
+	int ac = areas.size();
+	bool stopped = false;
+
+	if (pressure_coefficient < CMP_EPSILON && ac <= 0) {
```

Comment:
There could be an easy optimization here, by checking if any area has wind_force while checking gravity. If the wind force is always < epsilon and pressure is also < epsilon, this method doesn't have to run.